app/controllers/Application.scala

```diff
@@ -391,10 +394,12 @@ object Application extends Controller {
       case x => x + "/ws"
     }
     val prefix = if (request.secure) "wss" else "ws"
-    def ws_url(path:Option[String]=None) = s"""
-      |window.notebookWsUrl = function() {
-      |  return '$prefix:/'+window.location.host+'$wsPath${path.map(x => "/"+x).getOrElse("")}';
-      |};
+    def ws_url(path: Option[String] = None) = s"""
+                                                 |window.notebookWsUrl = function() {
```

Comment:
mmmmh wouldn't this screw the javascript definition? The problem I guess is the interpolated code that uses `{}` to internally compute the path :-/

To satisfy the code formatter, we would have to refactor it like:
```scala
def ws_url(path:Option[String]=None) = {
  val p = path.map(x => "/"+x).getOrElse("")
  s"""
    |window.notebookWsUrl = function() {
    |  return '$prefix:/'+window.location.host+'$wsPath$p';
    |};
  """.stripMargin.replaceAll("\n", " ")
}
```